dictionary.json

```diff
@@ -19491,7 +19491,7 @@
             "code": 455,
             "name": "Multiple-Services-Indicator",
             "vendorId": 0,
-            "type": "Unsigned32",
```

Comment:
This should be an Integer32. See https://github.com/node-diameter/node-diameter/pull/34#issuecomment-295723454